src/components/ActionButton.tsx

```diff
@@ -7,16 +7,20 @@ import Button from './Button'
 import Row, { RowProps } from './Row'
 import Tooltip from './Tooltip'
 
-const StyledButton = styled(Button)`
+const StyledButton = styled(Button)<{ useDisabledColor?: boolean }>`
```

Comment:
i'd probably not prefix this variable with `use` since thats more of a hooks convention 🤔 

you could also probably just pass in the `disabledBehavior` value here directly?

Reply:
i wanted to have as little logic in the CSS as possible, so i thought a boolean would be better. i will update the name though